watson-language/watson-lang.lua

```diff
@@ -0,0 +1,95 @@
+--[[
+
+# [Salesfors API](https://console.bluemix.net/docs/services/language-translator/getting-started.html) connector.
+
+### Authorization
+
+Authorization use [Public]().
+
+Watson API Explorer[Language Translator] (https://watson-api-explorer.mybluemix.net/apis/language-translator-v2)
+
+
+## Input parameters
+
+* watson_domain - e.g. `https://watson-api-explorer.mybluemix.net/language-translator/api/`.
```

Comment:
Could we make this optional? So if user don't submit watson_domain it will be pre-filled with this default value?